temba/orgs/models.py

```diff
@@ -2311,9 +2311,10 @@ def get_user_orgs(user, brand=None):
         return Org.objects.all()
 
     user_orgs = user.org_admins.all() | user.org_editors.all() | user.org_viewers.all() | user.org_surveyors.all()
+    not_suspended_orgs_ids = [org.id for org in user_orgs if not org.is_suspended()]
 
     if brand:
-        user_orgs = user_orgs.filter(brand=brand)
+        user_orgs = user_orgs.filter(brand=brand, id__in=not_suspended_orgs_ids)
```

Comment:
@srugano - I think you should just apply the suspended organization filter here on that method instead of also apply on https://github.com/greatnonprofits-nfp/rapidpro/pull/23/files#diff-beec431dcb947a4af6858c95c6d6179fR49 (so should be removed from `user_orgs_for_brand()`), another thing that I would like to ask you to change: this query to be filtered even if the `brand` is `None` so the method should be something like this:

```
def get_user_orgs(user, brand=None):

    if user.is_superuser:
        return Org.objects.all()

    user_orgs = user.org_admins.all() | user.org_editors.all() | user.org_viewers.all() | user.org_surveyors.all()
    not_suspended_orgs_ids = [org.id for org in user_orgs if not org.is_suspended()]

    user_orgs = user_orgs.filter(id__in=not_suspended_orgs_ids)

    if brand:
        user_orgs = user_orgs.filter(brand=brand)

    return user_orgs.filter(is_active=True).distinct().order_by("name")
```
That way, on unit tests you also should decrease in 1 the number of queries that you added.

Reply:
Sure. let me add that.